core/src/main/java/edu/softserve/zoo/util/ListUtils.java

```diff
@@ -0,0 +1,27 @@
+package edu.softserve.zoo.util;
+
+import edu.softserve.zoo.exceptions.ApplicationException;
+
+import java.util.List;
+
+/**
+ * <p>This utility class helps to work with lists</p>
+ *
+ * @author Taras Zubrei
+ */
+public class ListUtils {
```

Comment:
In my first version, It was using Collection. But then we changed PersistenceProvider.find() return to List and the util is developed for this case. So the class migrated to List from Collection.